frontend/src/app/search/page.tsx

```diff
@@ -25,6 +26,16 @@ interface ServerPageProps {
   searchParams: ServerSideSearchParams;
 }
 
+export function generateMetadata() {
+  // TODO: use the following for i18n const t = await getTranslations({ locale: params.locale });
+  const meta: Metadata = {
+    title: "Search Funding Opportunities | Simpler.Grants.gov",
+    description: "Try out our experimental search page.",
+  };
+
+  return meta;
+}
```

Comment:
@acouch it would be a little bit better to keep the `page.tsx` file slim . So for example moving this and beta strings out to their respective components.

Reply:
In NextJS the metadata is added to [a layout.js or static page.js file](https://nextjs.org/docs/app/building-your-application/optimizing/metadata#static-metadata). 

We could move the metadata to the layout in #1657 and #1361 . I noted it in the i18n ticket.